src/Raven.Studio/wwwroot/App/views/shell/about.html

```diff
@@ -172,9 +172,8 @@ <h4>Security</h4>
                                            attr: { title: replaceTooltip }">
                                 <i class="icon-replace"></i><span>REPLACE LICENSE</span>
                         </button>
-                        <button class="btn btn-info flex-grow"
+                        <button class="btn btn-warning flex-grow"
```

Comment:
How does it look like for a developer license with 3 options?

Reply:
image was attached in the issue:
https://issues.hibernatingrhinos.com/issue/RavenDB-17266#focus=Comments-67-127783.0-0